spec/support/shared/examples/metasploit/cache/module/rankable.rb

```diff
@@ -0,0 +1,9 @@
+shared_examples_for 'Metasploit::Cache::Module::Rankable' do |rank:|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.